src/rooster/_git.py

```diff
@@ -51,7 +53,7 @@ def get_commits_between(
         .id
         if second_version is not None
         # TODO: Lookup main branch
-        else repo.revparse_single("main").id
+        else repo.revparse_single(BRANCH_NAME).id
```

Comment:
Thanks for the pull requests!

Could we read this from the `Config` instead of using an environment variable?

?https://github.com/zanieb/rooster/blob/main/src/rooster/_config.py